resources/test/tests/functional/api-tests-1.html

```diff
@@ -190,12 +190,6 @@ <h1>Sample HTML5 API Tests</h1>
 
     async_test("test should timeout (fail) with the default of 2 seconds").step(function(){});
 
-    async_test("test should timeout (fail) with a custom set timeout value of 1 second",
```

Comment:
Can you revert these changes? We should delete the tests when the functionality itself is removed eventually.

Reply:
Reverted.